runelite-client/src/main/java/net/runelite/client/Notifier.java

```diff
@@ -120,6 +148,8 @@ private void sendLinuxNotification(
 		commands.add("notify-send");
 		commands.add(title);
 		commands.add(message);
+		commands.add("-i");
+		commands.add(notifyIconPath.toAbsolutePath().toString());
```

Comment:
This should probably be escaped similar to the message

Reply:
Escaped in most recent changes.